graphql/http_handler.go

```diff
@@ -30,5 +31,18 @@ func Handler(apiURL string) func(w http.ResponseWriter, r *http.Request) {
 		})
 		return errors.New("internal server error")
 	})
+
+	srv.SetErrorPresenter(func(ctx context.Context, err error) *gqlerror.Error {
+		fieldCtx := graphql.GetFieldContext(ctx)
+		queryPath := fieldCtx.Path()
+		args := fieldCtx.Args
+		grip.Error(message.Fields{
```

Comment:
In this case we can use message.WrapError() to wrap the error rather than setting it as "message" which would give us stack trace / other information if available.